perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/PerunBlImpl.java

```diff
@@ -1,30 +1,5 @@
 package cz.metacentrum.perun.core.blImpl;
 
-import cz.metacentrum.perun.core.api.Attribute;
```

Comment:
I believe we wanted to omit wildcard imports, because of clarity for non IDE like text editors. Even thou I believe we are not consistent in it.